src/modules/auth/actions.js

```diff
@@ -1,3 +1,5 @@
 export const types = {
 	AUTH_USERINFO: "SeaCat/AUTH_USERINFO",
+	UNAUTHORIZED_ITEM: "SeaCat/UNAUTHORIZED_ITEM",
```

Comment:
This is way to generic ... `SeaCat/Navigation/UNAUTHORIZED_ITEM` is likely more accurate